ci/docs.yml

```diff
@@ -1,10 +1,11 @@
+name: xdggs-docs
 channels:
   - conda-forge
 dependencies:
   - python=3.12
   - sphinx
-  - myst-parser
```

Comment:
is there a reason why you removed this? I guess I'm asking because I thought `myst-nb` was the equivalent of `nbsphinx` for the `myst` / `sphinx` ecosystem, and as far as I can tell it also has a dependency on `myst-parser`, so we'd install it anyways (but my knowledge of the `myst` ecosystem is spotty at best, so that might be the reason for my confusion).

Or was that because `myst_parser` was not part of the list of enabled extensions? In which case I'd probably just add that instead.

Either way the documentation builds fine, so this isn't a huge deal

Reply:
I think that this is redundant (since myst-nb requires parser). But in the extension list it is actully imperative to only **use one of the two**. Happy to revert this, but it doesnt really matter as you pointed out correctly.